source/debug.rst

```diff
@@ -0,0 +1,27 @@
+.. _debug-images:
+
+Debug
+=================
+
+For the cases when Pods are failing for some reason or just show abnormal behavior, 
+the Operator can be used with a special *debug image* of the Percona Server for
+MongoDB, which has the following specifics:
+
+* it avoids restarting on fail,
+* it contains additional tools useful for debugging (sudo, telnet, gdb, etc.),
+* it has debug mode enabled for the logs.
```

Comment:
I don't think we do this for PSMDB (adding additional debug mode when running mongod), please check with @qJkee or @hors. We could mention instead that packages with debug symbols are installed in this docker image.

Reply:
> I don't think we do this for PSMDB (adding additional debug mode when running mongod), please check with @qJkee or @hors. We could mention instead that packages with debug symbols are installed in this docker image.

Actually, i'm using `-vvv` flag, which adds more verbosity than usual output

Reply:
@qJkee ah, somehow I missed it, but in any case it would be good to have a line that in this image we have packages with included debug symbols installed

Reply:
@fiowro 
We can add to the list of additional tools - `mongodb debuginfo`
And instead of `it has debug mode enabled for the logs.`  use like `extra verbosity is added to mongodb daemon` 
@tplavcic  please correct if i'm wrong

Reply:
@qJkee looks good to me